cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDbException.java

```diff
@@ -76,7 +82,17 @@ public String getReason() {
     public String getMessage() {
         //improve the message with status code, error and reason if we have them
         return ((getStatusCode() > 0) ? getStatusCode() + " " : "") + super.getMessage()
+                + ((url != null) ? ": " + url : "")
```

Comment:
The current message looks like:
`404 Object Not Found: not_found: no_db_file`
I think the new message will look like:
`404 Object Not Found: https://examples.cloudant.com/mydb: not_found: no_db_file`

I wonder if use of the `:` in both the URL scheme and the message separators is a bit confusing.

Maybe we can just tack the URL on the end with "at"?
`404 Object Not Found: not_found: no_db_file at https://examples.cloudant.com/mydb`

Or maybe we should just make the message more human readable altogether, something like:
`404 Object Not Found at https://examples.cloudant.com/mydb. Error: not_found. Reason: no_db_file.`

Thoughts @smithsz @rhyshort